app/models/user.rb

```diff
@@ -101,6 +101,10 @@ def name
     name = first_name + " " + last_name
   end
 
+  def has_team_notifications_enabled?
+    true
```

Comment:
@birneamstiel FYI: Please make sure that this method is deleted when #301 gets implemented (and thus this method is automatically generated by Rails based on the equally-named ActiveRecord attribute).